packages/playwright-core/src/utils/stackTrace.ts

```diff
@@ -31,7 +31,12 @@ const CORE_DIR = path.resolve(__dirname, '..', '..');
 const CORE_LIB = path.join(CORE_DIR, 'lib');
 const CORE_SRC = path.join(CORE_DIR, 'src');
 const TEST_DIR_SRC = path.resolve(CORE_DIR, '..', 'playwright-test');
-const TEST_DIR_LIB = path.resolve(CORE_DIR, '..', '@playwright', 'test');
+let TEST_DIR_LIB = '';
```

Comment:
I find it strange that core depends on the @playwright/test assumptions. It would be better for @playwright/test to use semi-private core api to set its location instead. We already export stackTrace to test, so pwt could say addStackFilter(require.resolve('./')) in the index.ts fixtures file.